packages/rulesets/src/spectral/test/avoid-additional-properties.test.ts

```diff
@@ -173,6 +224,30 @@ test("AvoidAdditionalProperties should find no errors", () => {
           },
         },
       },
+      ManagedServiceIdentity: {
+        description: "Managed service identity (system assigned and/or user assigned identities)",
+        type: "object",
+        properties: {
+          principalId: {
+            readOnly: true,
+            format: "uuid",
+            type: "string",
+            description:
+              "The service principal ID of the system assigned identity. This property will only be provided for a system assigned identity.",
+          },
+          tenantId: {
+            readOnly: true,
+            format: "uuid",
+            type: "string",
+            description:
+              "The tenant ID of the system assigned identity. This property will only be provided for a system assigned identity.",
+          },
+          type: {
+            $ref: "src/spectral/test/resources/lro-provisioning-state-specified.json#/definitions/PrivateEndpointConnection",
```

Comment:
>src/spectral/test/resources/lro-provisioning-state-specified.json#/definitions/PrivateEndpointConnection [](http://example.com/codeflow?start=19&length=104)

does this path resolve locally?

Reply:
It is not giving error as unresolved as I set resolved to false on the condition, its not getting the additional properties from the PrivateEndpointConnection definition.